csrc/host_ir/executor.cpp

```diff
@@ -274,7 +275,19 @@ void HostIrEvaluator::handle(SetCurrentStream* set_current_stream) {
 }
 
 void HostIrEvaluator::handle(Synchronize* synchronize) {
-  getCUDAStream(synchronize->stream()).synchronize();
+  cudaStream_t current_stream =
+      c10::cuda::getCurrentCUDAStream(
+          static_cast<c10::DeviceIndex>(my_device_index_))
+          .stream();
+  cudaStream_t stream_to_sync = getCUDAStream(synchronize->stream()).stream();
+
+  cudaEvent_t event = {};
```

Comment:
```suggestion
  cudaEvent_t event;
```
if you still plan to use an event to sync

Reply:
The linter complains without "= {}", see https://github.com/NVIDIA/Fuser/actions/runs/12391828487/job/34589731271